types/react-auth-kit/react-auth-kit-tests.ts

```diff
@@ -0,0 +1 @@
+export {};
```

Comment:
An empty test file is no tests - either delete the file or add some tests, please.